packages/g/gd-sprest-bs.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gd-sprest-bs",
+  "filename": "dist/gd-sprest-bs-icons.min.js",
+  "description": "A JavaScript library to interact with the SharePoint REST API including Bootstrap components.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gunjandatta/sprest-bs"
+  },
+  "keywords": [
+    "sharepoint",
+    "spo",
+    "rest",
+    "bootstrap"
+  ],
+  "homepage": "https://dattabase.com/",
+  "npmName": "gd-sprest-bs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
```suggestion
        "*.@(js|ts)"
```
[Use a more specific glob.](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A.%40%28js%7Cts%29&matches=true&tests=%2F%2F%20Imported%20from%20gd-sprest-bs%20NPM%20package&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.css&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.min.css&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.structure.css&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.structure.min.css&tests=lib%2Fjquery-ui-1.12.1.custom%2Findex.html&tests=dist%2Fgd-sprest-bs-icons.js&tests=dist%2Fgd-sprest-bs-icons.min.js&tests=dist%2Fgd-sprest-bs.js&tests=dist%2Fgd-sprest-bs.min.js&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.js&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.min.js&tests=lib%2Fjquery-ui-1.12.1.custom%2Fexternal%2Fjquery%2Fjquery.js&tests=test.js&tests=lib%2Fjquery-ui-1.12.1.custom%2Fpackage.json&tests=package.json&tests=README.md&tests=src%2Fstyles.scss&tests=src%2Fwebparts%2Ftabs%2Fstyles.scss&tests=.%2F%40types%2Fcomponents%2Fdatetime.d.ts&tests=src%2Fcomponents%2Fdatetime.ts&tests=.%2F%40types%2Fcomponents%2Ffield.d.ts&tests=src%2Fcomponents%2Ffield.ts&tests=dist%2Fgd-sprest-bs.d.ts&tests=.%2F%40types%2Fwebparts%2Fhelper.d.ts&tests=src%2Fwebparts%2Fbase%2Fhelper.ts&tests=src%2Findex-icons.ts&tests=.%2F%40types%2Fcomponents%2Findex.d.ts&tests=.%2F%40types%2Findex.d.ts&tests=.%2F%40types%2Fwebparts%2Findex.d.ts&tests=src%2Fcomponents%2Findex.ts&tests=src%2Findex.ts&tests=src%2Fwebparts%2Fbase%2Findex.ts&tests=src%2Fwebparts%2Findex.ts&tests=src%2Fwebparts%2Flist%2Findex.ts&tests=src%2Fwebparts%2FlistFields%2Findex.ts&tests=src%2Fwebparts%2Fsearch%2Findex.ts&tests=src%2Fwebparts%2Ftabs%2Findex.ts&tests=src%2Fwebparts%2Ftaxonomy%2Findex.ts&tests=.%2F%40types%2Fcomponents%2FlistForm.d.ts&tests=src%2Fcomponents%2FlistForm.ts&tests=.%2F%40types%2Fcomponents%2FlistFormDialog.d.ts&tests=src%2Fcomponents%2FlistFormDialog.ts&tests=.%2F%40types%2Fcomponents%2FpeoplePicker.d.ts&tests=src%2Fcomponents%2FpeoplePicker.ts&tests=.%2F%40types%2Frest.d.ts&tests=src%2Frest.ts&tests=.%2F%40types%2Fwebparts%2Fwp.d.ts&tests=src%2Fwebparts%2Fbase%2Fwp.ts&tests=src%2Fwebparts%2Flist%2Fwp.ts&tests=src%2Fwebparts%2FlistFields%2Fwp.ts&tests=src%2Fwebparts%2Fsearch%2Fwp.ts&tests=src%2Fwebparts%2Ftabs%2Fwp.ts&tests=src%2Fwebparts%2Ftaxonomy%2Fwp.ts&tests=src%2Fwebparts%2Fbase%2FwpCfg.ts&tests=src%2Fwebparts%2Flist%2FwpCfg.ts&tests=src%2Fwebparts%2FlistFields%2FwpCfg.ts&tests=src%2Fwebparts%2Fsearch%2FwpCfg.ts&tests=src%2Fwebparts%2Ftaxonomy%2FwpCfg.ts&tests=.%2F%40types%2Fwebparts%2FwpList.d.ts&tests=.%2F%40types%2Fwebparts%2FwpListFields.d.ts&tests=.%2F%40types%2Fwebparts%2FwpSearch.d.ts&tests=.%2F%40types%2Fwebparts%2FwpTabs.d.ts&tests=.%2F%40types%2Fwebparts%2FwpTaxonomy.d.ts&tests=lib%2Fjquery-ui-1.12.1.custom%2FAUTHORS.txt&tests=lib%2Fjquery-ui-1.12.1.custom%2FLICENSE.txt&tests=pnpm-lock.yaml)

Reply:
This ^